bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,159 @@
+| host          | IP or string                    | IP address or hostname                                                  |   Yes     |
+| port          | integer                         | Port to reach Mycroft (default 8181)                                    |   No      |
+
+
+## Channels
+
+The Mycroft thing supports the following channels :
+
+
+| channel type id              | Item type | description                                                                                    |
+|------------------------------|-----------|------------------------------------------------------------------------------------------------|
+| listen                       | Switch    | Switch to ON when Mycroft is listening. Can simulate a wake word detection to trigger the STT  |
+| speak                        | String    | The last sentence Mycroft speaks                                                               |
+| utterance                    | String    | The last utterance Mycroft receive                                                             |
+| player                       | Player    | The music player Mycroft is currently controlling                                              |
+| volume_mute                  | Switch    | Mute the Mycroft speaker                                                                       |
+| volume_duck                  | Switch    | Duck the volume of the Mycroft speaker                                                         |
+| full_message                 | String    | The last message (full json) seen on the Mycroft Bus. Filtered by the messageTypes properties  |
+
+
+The channel 'full_message' has the following configuration available :
+
+| property      |  type                           | description                                                             | mandatory |
+|---------------|---------------------------------|-------------------------------------------------------------------------|-----------|
+| messageTypes  | List of string, comma separated | Only these message types will be forwarded to the Full Message Channel  |   No      |
+
+
+## Full Example
+
+A manual setup through a `things/mycroft.things` file could look like this:
+
+```java
+Thing mycroft:mycroft:myMycroft "Mycroft A.I." @ "Living Room" [host="192.168.X.X", port=8181] { 
+    Channels:
+        Type full-message-channel : Text [
+            messageTypes="message.type.1,message.type.4"
+        ]
+}
+```
+
+### Item Configuration
+
+The `mycroft.item` file :
+
+```java
+Switch myMycroft_mute                  "Mute"                      { channel="mycroft:mycroft:myMycroft:volume_mute" }
+Switch myMycroft_duck                  "Duck"                      { channel="mycroft:mycroft:myMycroft:volume_duck" }
+Dimmer myMycroft_volume                "Volume [%d]"               { channel="mycroft:mycroft:myMycroft:volume" }
```

Comment:
Done. By the way, should I click on the "resolve conversation" button to tell it's done, or should I let you do it to confirm your review ?
For now, I will let you click to resolve, but please tell me so if the "good practice" is to "resolve" it by myself.

Reply:
If you touch a line in the source where the comment is attached to, github automatically treats the discussion as resolved.

But i personally don't have any preferences on who clicks on the "resolved" button in case a discussion is left open. I mean in the end we have to find an agreement and when this has been achieved, I don't care on who presses a button ;)